model/host/db.go

```diff
@@ -426,6 +426,68 @@ func ByExpiringBetween(lowerBound time.Time, upperBound time.Time) db.Q {
 	})
 }
 
+// StateRunningTasks returns tasks documents that are currently run by a host and stale
+func FindStaleRunningTasks(cutoff time.Duration) ([]task.Task, error) {
+	pipeline := []bson.M{}
```

Comment:
nit: just do this as a literal thing. pipeline is always the same thing, you don't need to have the append functions.

Reply:
I separated them out for readability. I think this is slightly neater than putting them all in at once